spec/policies/user_policy_spec.rb

```diff
@@ -24,7 +24,7 @@
   end
 
   user_management_actions = [:edit?, :create?, :update?, :unlock?, :suspension?,
-    :cancel_email_change?, :resend_email_change?, :event_logs?]
+                             :cancel_email_change?, :resend_email_change?, :event_logs?]
```

Comment:
This is correct according to the main [Ruby style guide](https://github.com/bbatsov/ruby-style-guide#align-multiline-arrays), but really the elements should be on multiple lines (and wrapped in `%i()`, so I've added that.